telehealth-example/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.pubnub.components.example.getting_started">
```

Comment:
package should be removed from Manifest, since you've declared it in gradle file